tasks.py

```diff
@@ -89,13 +90,17 @@
 extra_dependencies = {
     'pyside': {'pyside'},
     'pyqt': {'pyqt'},
-    'wx': {'wxpython'},
+    # XXX once pyqt5 is available in EDM, we will want it here
+    'pyqt5': set(),
+    # XXX temporary workaround for bug in recent EDM wxpython build on OS X
+    'wx': {'wxpython' if sys.platform != 'darwin' else 'wxpython==3.0.2.0-3'},
```

Comment:
No, this is an issue with the EPD/EDM build of wxpython-3.0.2.0-4 OS X which is still the current version being served.  We can remove this if/when there is a 3.0.2.0-5 or higher build for OS X.